apps/accounts/templates/accounts/css/report.css

```diff
@@ -173,13 +152,15 @@ p {
 }
 
 .no-data {
+    color: #909399;
+    font: 400 14px/22px 'PingFang SC';
     text-align: center;
-    color: #6c757d;
-    padding: 2rem;
-    background: #f6f6f6;
-    border-radius: 8px;
-    margin: 1rem 0;
-    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
+    padding: 20px;
+    font-style: italic;
+    background-color: #f9f9f9;
+    border-radius: 4px;
+    margin: 10px 0;
+    width: 100%;
 }
 
 .new-accounts .section-header span {
```

Comment:
As of my last updated knowledge on September 1st, 2021, I do not have information about the latest changes and updates to this code. However, if you share more details or context regarding these files (e.g., specific sections like `.report-container`, `.header`, etc.), I might be able give tailored advice and recommendations based upon those specifics. Remember that it's important to check for recent security vulnerabilities such as cross-site scripting attacks while reviewing old versions.